go/storage/localdiskbackend.go

```diff
@@ -69,3 +72,44 @@ func (db *LocalDiskBackend) StoreKnownCertificateList(ctx context.Context, issue
 	}
 	return nil
 }
+
+// Write a line delimited list of serial numbers and reason codes to a text
+// file. Each line contains hex encoded binary data. The first (encoded) byte
+// in each line is the reason code. The remaining bytes are the serial number.
+func (db *LocalDiskBackend) StoreRevokedCertificateList(ctx context.Context, issuer types.Issuer,
+	serials []types.SerialAndReason) error {
+	path := filepath.Join(db.rootPath, issuer.ID())
+	if err := makeDirectoryIfNotExist(path); err != nil {
+		return err
+	}
+
+	fd, err := os.OpenFile(path, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, db.perms)
+	if err != nil {
+		return err
+	}
+	defer fd.Close()
+
+	writer := bufio.NewWriter(fd)
+	defer writer.Flush()
+
+	for _, s := range serials {
+		select {
```

Comment:
Is this really the right formatting for this? Or is this just a display issue because go uses tabs?

Reply:
gofmt is happy, and it doesn't look weird on my screen... :shrug:

Reply:
I just meant that the body of the `select` doesn't increase indentation, but I guess that's the style?

Reply:
ah, yeah that's the style.